activation/activation.go

```diff
@@ -312,33 +306,36 @@ func (b *Builder) PublishActivationTx(epoch types.EpochId) error {
 	// we've completed the sequential work, now before publishing the atx,
 	// we need to provide number of atx seen in the epoch of the positioning atx.
 	posEpoch := b.posLayerID.GetEpoch(b.layersPerEpoch)
-	activeIds, err := b.activeSet.ActiveSetSize(posEpoch)
-	if err != nil && !posEpoch.IsGenesis() {
-		return err
-	}
-	view, err := b.mesh.GetOrphanBlocksBefore(b.mesh.LatestLayer())
-	if err != nil {
-		return err
+	pubEpoch := b.challenge.PubLayerIdx.GetEpoch(b.layersPerEpoch) // can be 0 in first epoch
+	targetEpoch := pubEpoch + 1
+
+	viewLayer := pubEpoch.FirstLayer(b.layersPerEpoch)
+	var view []types.BlockID
+	if viewLayer > 0 {
```

Comment:
else...?

Reply:
if the `viewLayer` is 0 we should keep the view empty, as there are no blocks prior to this layer. First epoch ATXs will have an empty view and that's fine.